MAINTENANCE.md

```diff
@@ -84,16 +84,19 @@ This workflow is ran to publish wheels to PyPI and TestPyPI (for testing only).
 Archives will be pushed to TestPyPI on every commit to the *master* branch and
 tagged releases, and to PyPI for tagged releases only.
 
+6. `continuous-documentation.yml` (Deploy documentation to Vercel for preview)
+
+This workflow builds and deploys the documentation in Pull Requests to Vercel,
+to make it easier to preview documentation changes.
 
 ## Continuous Documentation
 
-We use the [Vercel for GitHub](https://github.com/apps/vercel) App to preview changes
-made to our documentation website every time we make a commit in a pull request.
-The service has a configuration file `vercel.json`, with a list of options to
-change the default behaviour at https://vercel.com/docs/configuration.
-The actual script `package.json` is used by Vercel to install the necessary packages,
-build the documentation, copy the files to a 'public' folder and deploy that to the web,
-see https://vercel.com/docs/build-step.
+We use the [Vercel](https://vercel.com/) service and the
+[vercel-action](https://github.com/marketplace/actions/vercel-action) action to
+preview changes made to our documentation website every time we make a commit
+in a pull request. The workflow `continuous-documentation.yml` builds and
+deploys the documentation to Vercel and create a comment with the latest URL
+of deployment preview.
```

Comment:
The bot's comment seems to only point to a single URL (e.g. https://pygmt-git-vercel-action-gmt.vercel.app in this PR) and now a unique one.

```suggestion
deploys the documentation to Vercel. The vercel bot will automatically make a
comment with a URL to preview the deployed documentation for that pull request.
```